main.tf

```diff
@@ -129,6 +129,10 @@ resource "aws_iam_role" "rds_enhanced_monitoring" {
   assume_role_policy = data.aws_iam_policy_document.monitoring_rds_assume_role.json
 
   permissions_boundary = var.permissions_boundary
+  
+  tags = merge(var.tags, {
+    Name = local.name
```

Comment:
I'm encountering an `InvalidInput: Duplicate tag keys found` when upgrading to 2.20+.

If `name` is already provided in `var.tags`, this would throw the duplicate error. Could this simply be `tags = var.tags`?

Reply:
Opened this as https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/issues/189